.release-version

```diff
@@ -1 +1 @@
-release-3.3.0
\ No newline at end of file
+3.3.1
```

Comment:
I just noticed this (it's the same comment for LabWhere) but I think this change is a minor, not really a bug fix but we can keep it as is.